MachUpskillingFY17/JabbR-Core
# Account controller integration

Use Cases to Test:
 - Register a new user
 - Create a room and send a message in that room
 - Log out (you might have to restart the project to avoid the self-referencing loop error in issues #126 and #129)
 - Log the user back in and check to make sure you can see the old messages in the room
 - Change the user's name and password
 - Log out
 - Log back in with the new name and password
 - Log out
 - Log back in one more time but this time say you "forgot your password"
 - Verify you got an email with a new password and can log in
 - Log out
 - Register a new user using Google authentication
 - Register a new user using Facebook authentication

Try to enter invalid entries in each form and verify all validation messages are working properly. 
Register a couple users and verify old messages from other users display in the room.
